gradle/testing/randomization/policies/solr-tests.policy

```diff
@@ -226,4 +226,8 @@ grant {
   permission java.io.FilePermission "${gradle.worker.jar}", "read";
   // Allow reading from classpath JARs (resources).
   permission java.io.FilePermission "${gradle.user.home}${/}-", "read";
+  // Allow reading from local Lucene development repository, if used.
+  permission java.io.FilePermission "${lucene-dev-repo.dir}${/}-", "read";
+  // Allow reading jars from mavenLocal repository
+  permission java.io.FilePermission "${user.home}${/}.m2${/}repository${/}org${/}apache${/}lucene${/}-", "read";
```

Comment:
One side-effect of needing to set the `lucene-dev-repo.dir` property so that it's accessible in the security policy here is that if that property is left _unset_, the entire associated `permission...` line is ignored. I'm sure it's not a big deal one way or another, but we'd have the option of taking an analogous approach `mavenLocal`. A couple of benefits, maybe more theoretical than practical:
1. the additional permission line would only have an effect in the cases where it's relevant
2. the construction and addition of the mavenLocal path could be kept within the build tool, which seems perhaps cleaner than building it directly (and unconditionally) in the java security policy file

Reply:
Correct, if a property is unset, the line is ignored. To be honest, I don't think it's a big problem even if we opened up the entire .m2 repository for read-access... I don't need it has to be so fine-grained and I don't have a problem with it. I'm for occam's razor - less code doing the same thing is better.

Reply:
I'm all for simplicity, and certainly defer to you wrt security policy stuff. I guess I was reading the purpose of the security policy file as: to keep things as locked-down as possible to avoid any local filesystem dependencies that aren't explicitly declared. Either way, as I mentioned I agree this is unlikely to present a practical problem. Maybe a note about why/when this access is needed then? (another benefit of setting a property specific to the permission is to give a clue that can be used to trace the purpose of the permission without resorting to `git blame`, etc. ...)